modules/n1ql/pages/n1ql-language-reference/delete.adoc

```diff
@@ -249,4 +269,44 @@ include::example$dml/delete-filter.n1ql[]
 ----
 include::example$dml/delete-filter.jsonc[]
 ----
+====
+
+.Delete with LIMIT and OFFSET
+====
+This example deletes a subset of the airlines with a country of "France'.
+First, you query to get a list of the airlines in France.
+
+[source,sqlpp]
+----
+include::example$dml/delete-offset-before.n1ql[]
+----
+
+There are 21 documents in this collection with `country="France"`, shown for this example in Table view.
+
+.Results
+[source,text]
+----
+include::example$dml/delete-offset-before.jsonc[]
+----
+
+<.> The 10th document's id.
+<.> The 20th document's id.
+
+Next, you specify that you want to delete up to 10 documents, after skipping the first 10.
+
+[source,sqlpp]
+----
+include::example$dml/delete-offset.n1ql[]
+----
+
+There are now 11 documents in this collection with `country="France"`, shown for this example in Table view.
+
+.Results
+[source,text]
+----
+include::example$dml/delete-offset.jsonc[]
+----
+
+<.> Documents with the first 10 ids&mdash;the offset&mdash;remain in the airline collection.
+<.> After deleting 10 documents&mdash;the limit&mdash;1 more document remains in the collection.
```

Comment:
Spaces on either side of em dash? I foresee a style guide discussion in the future.

Also, while `&mdash;` works, you can use the more idiomatic `--` in AsciiDoc.

```suggestion
<.> Documents with the first 10 ids -- the offset -- remain in the airline collection.
<.> After deleting 10 documents -- the limit -- 1 more document remains in the collection.
```

Reply:
no spaces, -- per Slack convo